tests/db_functions/test_trim.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import CharField
+from django.db.models.functions import LTrim, RTrim, Trim
+from django.test import TestCase
+
+from .models import Author
+
+
+class TrimTests(TestCase):
+    def test_trim(self):
+        Author.objects.create(name='  John ', alias='j')
+        Author.objects.create(name='Rhonda', alias='r')
+        authors = Author.objects.annotate(
+            ltrim=LTrim('name'),
+            rtrim=RTrim('name'),
+            trim=Trim('name'),
+        )
+        self.assertQuerysetEqual(
+            authors.order_by('alias'), [
+                ('John ', '  John', 'John'),
+                ('Rhonda', 'Rhonda', 'Rhonda'),
+            ],
+            lambda a: (a.ltrim, a.rtrim, a.trim)
+        )
+        for function in (LTrim, RTrim, Trim):
+            with self.subTest(function=function):
+                with self.assertRaisesMessage(
```

Comment:
Add a separate test method, and:
```
msg = "..."
with self.assertRaisesMessage(TypeError, msg):
```
By the way, what's your thinking about these tests since it's the default behavior of `Transform`?

Reply:
Maybe, you're right :thinking:. This doesn't increase a test coverage, so I'm fine to get rid of these tests :+1:.